src/net.cpp

```diff
@@ -1734,7 +1734,15 @@ void ThreadOpenConnections()
                 LogPrint("net", "Making feeler connection to %s\n", addrConnect.ToString());
             }
 
-            OpenNetworkConnection(addrConnect, (int)setConnected.size() >= std::min(nMaxConnections - 1, 2), &grant, NULL, false, fFeeler);
+            int64_t nStartTime = GetTimeMicros();
+
+            bool fRet = OpenNetworkConnection(addrConnect, (int)setConnected.size() >= std::min(nMaxConnections - 1, 2), &grant, NULL, false, fFeeler);
```

Comment:
nit: "ret" seems to be the standard so far, with 69 instances, compared to only 10 instances of "fRet" in this project.